sdk/canton/community/ledger/ledger-common/src/main/scala/com/digitalasset/canton/metrics/Metrics.scala

```diff
@@ -8,18 +8,21 @@ import com.daml.metrics.api.MetricHandle.LabeledMetricsFactory
 import com.daml.metrics.api.MetricName
 import com.daml.metrics.api.opentelemetry.OpenTelemetryMetricsFactory
 import com.daml.metrics.grpc.DamlGrpcServerMetrics
+import com.typesafe.scalalogging.LazyLogging
 import io.opentelemetry.api.metrics.Meter
 import io.opentelemetry.sdk.metrics.SdkMeterProvider
 
-object Metrics {
+object Metrics extends LazyLogging {
+
+  lazy val KNOWN_METRICS = Set("booh")
```

Comment:
Do we wanna keep this?

Reply:
Yes. This is the same as for the canton metrics above. I'll finish that in the Canton PR.